static/sass/_patterns_featured-snaps.scss

```diff
@@ -0,0 +1,18 @@
+@mixin snapcraft-p-featured-snaps {
+  .p-featured-snaps {
+    @extend %image-grid;
+
+    .p-featured-snaps__item {
+      // by default (on mobile) use col-3 width (4 items in a row)
```

Comment:
Shouldn't it say `by default (on desktop)`? It seems like that's what the code is doing.